scalding-beam/src/test/scala/com.twitter.scalding.beam_backend/BeamBackendTests.scala

```diff
@@ -0,0 +1,49 @@
+package com.twitter.scalding.beam_backend
+
+import com.twitter.scalding.{Config, Execution, TextLine, TypedPipe}
+import org.apache.beam.sdk.Pipeline
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.scalatest.{BeforeAndAfter, FunSuite}
+
+class BeamBackendTests extends FunSuite with BeforeAndAfter {
+
+  private val master = "local[2]"
+  private val appName = "beam-backend-tests"
+
+  private var pipeline: Pipeline = _
+
+  before {
+    val options = PipelineOptionsFactory.create()
+    pipeline = Pipeline.create(options)
+  }
+
+  after {
+    pipeline = null
+  }
+
+  test("test") {
+    val pref = "scalding-beam/src/test/scala/com.twitter.scalding.beam_backend/BeamBackendTests.scala"
+    val outRoute1 = pref + "out1"
+    val outRoute2 = pref + "out2"
+
+    val pipe1 = TypedPipe
+      .from(0 to 100)
+      .map(x => (x, x))
+      .mapValues(x => x * 3)
+      .filterKeys(x => x % 2 == 0)
+      .map(_.toString())
+    val ex1 = pipe1.writeExecution(TextLine(outRoute1))
+
+    val pipe2 = TypedPipe
+      .from(1 to 20)
+      .map(x => (x, x))
+      .flatMapValues(x => 1 to x)
+      .map(_.toString())
+    val ex2 = pipe2.writeExecution(TextLine(outRoute2))
+
+    val bmode = BeamMode.default(pipeline)
+    val ex3 = Execution.zip(ex1, ex2)
+    ex3.waitFor(Config.empty, bmode).get
```

Comment:
there is no assertion here. Can we check something? I guess that's todo.

One thing that would be really nice is eventually implement this test:

https://github.com/twitter/scalding/blob/b6d465f6fc308d66e5a53c62662d5860846d8dc4/scalding-core/src/test/scala/com/twitter/scalding/typed/memory_backend/MemoryTest.scala#L74

Reply:
We are just implementing SimpleWrite for the moment, I will add assertions in next steps when getForced and getIterable are implemented